webextension/css/popup.css

```diff
@@ -636,6 +647,32 @@ span ~ .panel-header-text {
   padding-inline-start: 16px;
 }
 
+#edit-sites-assigned {
+  flex: 1;
+}
+
+#edit-sites-assigned h3 {
+  font-size: 14px;
+  font-weight: normal;
+  padding-block-end: 5px;
+  padding-inline-end: 16px;
+  padding-inline-start: 16px;
+}
+
+#edit-sites-assigned table td {
+  display: flex;
+  padding-inline-end: 16px;
+  padding-inline-start: 16px;
```

Comment:
I don't think we support `padding-inline` and `padding-block` shorthands, will raise a bug about them if there isn't one.

Yeah they were added to the spec in April I doubt we support them I certainly couldn't get them to work: https://github.com/w3c/csswg-drafts/issues/1279

Will raise all the same. Longer term we could neaten this with `logical` prefixes or changing the linting rule to accept all 4 dimensions being the same like `margin: 0;` which is super boilerplate with our linting right now.